src/main/java/org/broadinstitute/consent/http/service/dao/DarCollectionServiceDAO.java

```diff
@@ -73,11 +73,14 @@ public List<String> createElectionsForDarCollection(User user, DarCollection col
           //    4. Create an RP Election
           //    5. Create member votes for rp election
           //        5a. Chair Vote for chair
-          collection.getDars().values().forEach(dar -> {
-            dar.getDatasetIds().forEach(datasetId -> {
+
+          // Only take actions on the most recent DAR/Progress report in the collection
+          // This means a chair cannot reopen an election in any other DAR in the collection besides
+          // the most recently submitted progress report.
+          dataAccessRequest.getDatasetIds().forEach(datasetId -> {
```

Comment:
In general I like forms like `forEach()` but in cases like this, it doesn't seem to add much and causes the code to be indented further than a `for` loop, so I would use `for` instead here
```suggestion
          for (var datasetId: dataAccessRequest.getDatasetIds()) {
```

Reply:
With the variable renamed to `dar` I think I prefer the forEach.